google/ggrc-core
# [Matrix view] GGRC-8747 Process “Save answers” button on “Bulk Complete” view

# Dependencies

This PR is `on hold` until the dependencies are merged:

- [x] #10828 


# Issue description

Process “Save answers” button on “Bulk Complete” view

# Steps to test the changes


1. Open Bulk completion mode.
2. Make any changes for attributes values.
3. Click Save Answers button.
4. Verify that floating message is shown and new attributes values are saved.

# Solution description

1. Added onSaveAnswersClick method for “Save answers” button.
2. Dispatched `refreshItemsList` event on assessment state change to track updates for Bulk completion button.
3. Fixed tooltip placement for Create assessments button.
4. Fix `beforeLoadItems` handler for assessment tree-actions.
5. Added soft info for Bulk completion grid.

# Sanity checklist




# PR Review checklist
